src/main/java/org/generationcp/middleware/dao/germplasmlist/GermplasmListDataSearchDAO.java

```diff
@@ -6,6 +6,7 @@
 import org.generationcp.middleware.api.germplasmlist.data.GermplasmListDataViewModel;
 import org.generationcp.middleware.api.germplasmlist.data.GermplasmListStaticColumns;
 import org.generationcp.middleware.dao.GenericDAO;
+import org.generationcp.middleware.dao.util.DAOQueryUtils;
```

Comment:
@lik-83 is this import required?

Reply:
It's required because I moved DAOQueryUtils to another package.